docs/CHANGELOG.asciidoc

```diff
@@ -62,6 +63,35 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.13.2]]
+== {kib} 8.13.2
+
+The 8.13.2 release includes the following bug fixes.
+
+[float]
+[[fixes-v8.13.2]]
+=== Bug Fixes
+Canvas::
+* Fixes text settings to be honored in Canvas markdown elements ({kibana-pull}179948[#179948]).
+Discover::
+* Fixes keyboard navigation for search input on the document viewer flyout ({kibana-pull}180022[#180022]).
+Elastic Security::
+For the Elastic Security 8.13.2 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Fixes having to wait ten minutes after agent upgrade if agent cleared watching state ({kibana-pull}179917[#179917]).
+* Fixes using the latest compatible version in K8's manifest instead of latest available version ({kibana-pull}179662[#179662]).
```

Comment:
```suggestion
Fixes using the latest available version in K8's manifest instead of the latest compatible version ({kibana-pull}179662[#179662]).
```
This should be reversed, based on Kyle's comment [here](https://github.com/elastic/ingest-docs/pull/1003#discussion_r1555739707).

Reply:
Thanks @kilfoyle, I totally missed that!